src/com/company/Main.java

```diff
@@ -1,12 +1,66 @@
 package com.company;
 
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 import java.util.Random;
 
 public class Main {
 
     public static void main(String[] args) {
 
+        // ������� ��������� �����
+        Random random = new Random();
+        List<Integer> numbers = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            numbers.add(random.nextInt(100));
+        }
+        System.out.println("random numbers: " + numbers.toString());
+
+        // ��������� ������� ���������� �������, � ��������� ���� ��� ������
+        Integer sum = numbers.stream()
+                .filter(integer -> integer %2==0)
```

Comment:
Reformat your code. Please, check your code before each commit operation, to avoid formatting issues